src/environments/environment.common.ts

```diff
@@ -30,5 +30,9 @@ export const commonEnv = {
   snapshotRoles: [
     'snapshot1.roles.consortiapool.apps.energyweb.iam.ewc',
     'snapshot2.roles.consortiapool.apps.energyweb.iam.ewc',
+    'snapshot3.roles.consortiapool.apps.energyweb.iam.ewc',
+    'snapshot4.roles.consortiapool.apps.energyweb.iam.ewc',
+    'silverpool.roles.consortiapool.apps.energyweb.iam.ewc',
+    'goldpool.roles.consortiapool.apps.energyweb.iam.ewc'
```

Comment:
@dawidgil do you see any issue with my having updated these roles in `environment.common`?

Reply:
No, this is only for dev and local env. If you want push something to prod, you should look on environment.prod.ts. Anyway i think this environment.common.ts should be removed as for me it is confusing :/